idea-compatibility/pom.xml

```diff
@@ -16,22 +16,22 @@
         <dependency>
             <groupId>com.intellij</groupId>
             <artifactId>openapi</artifactId>
-            <version>${idea.version}</version>
+            <version>IC-139.1603.1</version>
```

Comment:
Without this change "idea-compatibility" is building with right version IC-139.1603.1 but build of "idea-impl-139:3.13" fails.

Maven output:
[INFO] ------------------------------------------------------------------------
[INFO] Building idea-impl-139:3.13 3.13
[INFO] ------------------------------------------------------------------------
Downloading: https://repo.maven.apache.org/maven2/com/intellij/openapi/IC-129.239/openapi-IC-129.239.pom

[WARNING] The POM for com.intellij:openapi:jar:IC-129.239 is missing, no dependency information availableDownloading: https://repo.maven.apache.org/maven2/com/intellij/plugins/tasks/tasks-api/IC-129.239/tasks-api-IC-129.239.pom

[WARNING] The POM for com.intellij.plugins.tasks:tasks-api:jar:IC-129.239 is missing, no dependency information availableDownloading: https://repo.maven.apache.org/maven2/com/intellij/util/IC-129.239/util-IC-129.239.pom

[WARNING] The POM for com.intellij:util:jar:IC-129.239 is missing, no dependency information availableDownloading: https://repo.maven.apache.org/maven2/com/intellij/extensions/IC-129.239/extensions-IC-129.239.pom

[WARNING] The POM for com.intellij:extensions:jar:IC-129.239 is missing, no dependency information availableDownloading: https://repo.maven.apache.org/maven2/com/intellij/util/IC-129.239/util-IC-129.239.jar
Downloading: https://repo.maven.apache.org/maven2/com/intellij/extensions/IC-129.239/extensions-IC-129.239.jar